### DISCUSSION TITLE
Distance not reading correct.

### DISCUSSION THREAD

**Discussion by splasheye** [ESPresense/ESPresense#105] [Support]
Hi, love this project so far. I only have the ESPresense running on one ESP32 board at this time. I do not have the board in any case just sitting on my floor at this time. I am using my Galaxy S10 running Home Assistant companion app with BLE turned on. If I have my phone about 1meter 3 feet away it would display it in HA as 7 to 8. As far as I know the distance is displayed in meters. Is there anyway to get this to be more accurate?

**Comment by DTTerastar** [comment]
I believe it's related to this: https://github.com/home-assistant/android/issues/1715#issuecomment-950034433

**Reply by splasheye** [reply]
Thanks for the comment, will look at this information.
